homeassistant/components/comfoconnect/fan.py

```diff
@@ -72,50 +77,23 @@ def _handle_update(self, value):
         self.schedule_update_ha_state()
 
     @property
-    def should_poll(self) -> bool:
-        """Do not poll."""
-        return False
-
-    @property
-    def unique_id(self):
-        """Return a unique_id for this entity."""
-        return self._ccb.unique_id
-
-    @property
-    def name(self):
-        """Return the name of the fan."""
-        return self._name
-
-    @property
-    def icon(self):
-        """Return the icon to use in the frontend."""
-        return "mdi:air-conditioner"
-
-    @property
-    def supported_features(self) -> int:
-        """Flag supported features."""
-        return SUPPORT_SET_SPEED
-
-    @property
-    def percentage(self) -> int | None:
+    def percentage(self):
         """Return the current speed percentage."""
         speed = self._ccb.data.get(SENSOR_FAN_SPEED_MODE)
         if speed is None:
             return None
         return ranged_value_to_percentage(SPEED_RANGE, speed)
 
     @property
-    def speed_count(self) -> int:
+    def speed_count(self):
         """Return the number of speeds the fan supports."""
         return int_states_in_range(SPEED_RANGE)
 
-    def turn_on(
-        self, speed: str = None, percentage=None, preset_mode=None, **kwargs
-    ) -> None:
+    def turn_on(self, speed=None, percentage=None, preset_mode=None, **kwargs):
         """Turn on the fan."""
         self.set_percentage(percentage)
 
-    def turn_off(self, **kwargs) -> None:
+    def turn_off(self, **kwargs):
```

Comment:
Instead of removing typing, adding it to complete it would be better

```suggestion
    def turn_off(self, **kwargs: Any) -> None:
```